contracts/policies/PowerEventReplacement.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.4.11;
+
+// #ONLY CHANGED startCollection FUNCTION TO ACT AS A REPLACEMENT FOR OLD POWER EVENT
+
+import '../policies/PowerEvent.sol';
+
+contract PowerEventReplacement is PowerEvent {
+
+  // #KEEP ALL THE OTHER PARAMS SAME AS oldPowerEvent except for where *_controllerAddr = _nextControllerAddress, *_discount = 1000000
+  function PowerEventReplacement(address _controllerAddr, uint256 _startTime, uint256 _minDuration, uint256 _maxDuration, uint256 _softCap, uint256 _hardCap, uint256 _discount, uint256 _amountPower, address[] _milestoneRecipients, uint256[] _milestoneShares)
+    PowerEvent(_controllerAddr, _startTime, _minDuration, _maxDuration, _softCap, _hardCap, _discount, _amountPower, _milestoneRecipients, _milestoneShares) {
+  }
+
+  // #CHANGED THIS FUNCTION TO REFLECT OLD PowerEvent VARIABLES
+  function startCollection() isState(EventState.Waiting) {
+    // check time
+    require(now > startTime);
+    // assert(now < startTime.add(minDuration));
+    // read initial values
+    var contr = Controller(controllerAddr);
+    powerAddr = contr.powerAddr();
+    nutzAddr = contr.nutzAddr();
+    initialSupply = 2399896170149257466012; //initialSupply as per old Power Event
```

Comment:
@bkcninja where these came from?

For current PowerEvent these are:
0 and 3000000000000000 respectively

Reply:
I must be missing something

Reply:
sorry, I've been looking into wrong PowerEvent. Ignore me